cluster-stamp.json

```
@@ -738,7 +740,7 @@
         {
             "type": "Microsoft.Network/privateEndpoints",
             "apiVersion": "2020-05-01",
-            "name": "nodepool-to-acr",
+            "name": "[concat('nodepool-to-acr', '-', variables('vnetName'), '-', variables('subnetName'))]",
```

```suggestion
            "name": "[concat('acr-to-', variables('vnetName'))]",
```

After seeing it, I think actually would even be better. What do you think?  No need for additional (duplicated) subnet name variable, and is distinguished enough in our deployment.  Also, we'll be updating which subnet this targets soon (will be a new subnet instead of `snet-nodepools` in the future), so keeping that part out would be good.

---

The suggestions looks fine to me.